consensus/src/authority/mod.rs

```diff
@@ -308,9 +308,10 @@ mod tests {
                     nullify_retry: Duration::from_secs(10),
                     fetch_timeout: Duration::from_secs(1),
                     activity_timeout,
-                    max_fetch_count: 1,
+                    max_fetch_count: 4,
```

Comment:
When a notarization/nullification is missing at a parent, requests for both notarization/nullification will be sent. If a notarization doesn't exist and the limit is 1, we will never resolve.